dfastmi/batch/core.py

```diff
@@ -483,21 +483,22 @@ def _get_root_dir(rootdir: Path) -> Path:
     return rootdir
 
 
-def count_discharges(discharges: Vector) -> int:
+def count_discharges(apply_q: Tuple[bool, bool, bool]) -> int:
```

Comment:
If this is a legacy method move to legacy.py. Do not do this in this issue (not in scope) but create a new issue to cleanup this file for improved maintenance.

Reply:
Included in issue DFAST-237.